documentation/source/simulator_support.rst

```diff
@@ -91,7 +91,7 @@ To run cocotb with Verilator, you need ``verilator`` in your PATH.
 
 .. versionadded:: 1.3
 
-.. versionchanged:: 1.5 Improved cocotb support and greatly improved performance when using a higher time precision. Verilator 4.106 or later is required.
```

Comment:
I think removing this disclaimer is not a good idea. I also think any improvement to the Verilator docs include a strong warning about Verilator support for cocotb being unstable.

Reply:
Although it is repeated from the note above, having it in the `versionchanged` tag tracks the history of supported versions.

Reply:
Although it is repeated from the note above, having it in the `versionchanged` tag tracks the history of supported versions. Not critical though.

Reply:
Agreed, especially since there is this `changes` Sphinx builder which can extract these comments.

Reply:
Reinstated.